WordPress/Classes/Models/ReaderTeamTopic.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 
 @objc open class ReaderTeamTopic: ReaderAbstractTopic {
     @NSManaged open var slug: String
+    @NSManaged open var organizationID: NSNumber
```

Comment:
In the datamodel, it looks like this is set to use the scalar value. Maybe we could just make this an `Int` vs an `NSNumber`?  This would let us avoid having to constantly call .intValue later.
Fwiw, in the data model properties we could also uncheck "optional" since a default of 0 is provided.

Reply:
I'm probably being thick here, but I don't see "optional" on the Attributes. What am I missing?

<img width="406" alt="Screen Shot 2020-12-14 at 7 08 58 PM" src="https://user-images.githubusercontent.com/1816888/102158999-dc9bec80-3e3f-11eb-9a1e-28606edf3db0.png">

Reply:
It's in the sidebar's details for the organizationID attribute.  For me, sometimes Xcode is a little fussy. If the side panel doesn't show the attribute properties when organizationID is selected try tapping to the side.  It's an annoying bug that's been around for a while. :/ 
![Screen Recording 2020-12-15 at 9 32 07 AM 2020-12-15 09_35_33](https://user-images.githubusercontent.com/1435271/102236280-ee6da600-3eb8-11eb-90ee-0e4b7cf5d673.gif)

Reply:
> It's in the sidebar's details for the organizationID attribute.

Wowza. I _was_ being thick. I keep the sidebar closed, and it just didn't occur to me to look there. Sorry for the trouble! I've unchecked `optional` for the three instances of `organizationID` I added it to recently (`ReaderTeamTopic`, `ReaderSiteTopic`, `ReaderPost`).